speech/cloud-client/src/main/java/com/example/speech/QuickstartSample.java

```diff
@@ -33,39 +33,43 @@
 import java.util.List;
 
 public class QuickstartSample {
+
+  /**
+   * Demonstrates using the Speech API to transcribe an audio file.
+   */
   public static void main(String... args) throws Exception {
     // Instantiates a client
-    SpeechClient speech = SpeechClient.create();
+    try (SpeechClient speechClient = SpeechClient.create()) {
```

Comment:
we use speech everywhere else, why not keep it consistent.  (don't really care which one you choose, but speech is shorter.